environments/swiss/public.yml

```diff
@@ -77,7 +77,7 @@ report_builder_add_on_email: sales@dimagi.com
 eula_change_email: eula-notifications@dimagi.com
 contact_email: info@dimagi.com
 soft_assert_email: commcarehq-ops+soft_asserts@dimagi.com
-
+check_s3_backups_email: commcarehq-ops+root@dimagi.com,httu.admin@swisstph.ch
```

Comment:
can you add this parameter to the backups documentation?

Reply:
Yes this is a great suggestion. Thanks